README.md

```diff
@@ -200,9 +211,77 @@ Security Team will need to update the project release node on backdropcms.org to
 inidciate that a release is a security release.
 
 
+The Backdrop Bug Squad
+----------------------
+
+The `Backdrop Bug Squad` consistes of trusted members of the backdrop-contrib
+group who will help other contributors stay on top of minor bug fixes and User
+Interface improvements.
+
+Responsibilities include:
+- Reviewing Pull Requests for eligibility.
+- Merging eligible Pull Requests.
+- Communicating with existing maintainers.
+
+Fixes eligible inlude:
+- Minor Bugs (Fatal errors, PHP notices, validation errors, etc).
+- Minor User Interface changes (typos, spelling, grammar, terminology, etc).
+- Any packaging issue preventing the project from being downloaded and installed
+  via the Project Browser.
+- Any change that could be accepted after a feature freeze.
+
+Fixes NOT eligible include:
+- Any new feature.
+- Any major code change or addition.
+
+Any eligable Pull Request must be tagged as "Reviewed and Tested By the
+Community" for a period of at least 2 weeks before it can be merged.
+
+You may apply to become a member of the Backdrop Bug Squad. Please
+[create an issue in this repository](https://github.com/backdrop-ops/contrib/issues/new)
+requesting to join.
+
+
 Abandoned Projects
 ------------------
 
+An abandoned project is one with no maintainer listed in the `README.md` file.
+
+The ony changes that can be made to an abandoned project are security fixes, amd
+Pull Rrequests that are merged by the `Backdrop Bug Squad` (see above).
+
+An abandoned project will not have any new releases other than a secutity
+release issued by the `Backdrop Security Team` or a bug-fix release issued
+by the `Backdrop Bug Squad`. A project must have a current maintainer listed in
+the `README.md` file to have a new release containing new features, or any
+substantial changes.
+
+
+Determining Abandoned Projects
+------------------------------
+
+If you believe that a project has been abandoned but there is a maintainer
+listed in the `README.md` file, you can open an issue asking the maintainer if
+the project is abadoned.
+
+- If the maintainer responds within 2 weeks, the project has not been abandoned.
+- If the maintainer does NOT respond within 2 weeks, you can [create an issue in this repository](https://github.com/backdrop-ops/contrib/issues/new)
+  to request that the maintainer's name be removed from the `README.md` file.
+  Please include a link to the issue you filed for the project.
+- Please understand that removing all maintainers will hault all forward
+  progress for the project. It would be better to...
```

Comment:
...removing all maintainers will halt all forward progress for the project, as no new releases will be allowed until there is at least one maintainer.

Reply:
added.